epsagon/trace.py

```diff
@@ -1,6 +1,7 @@
 """
 Trace object holds events and metadata
 """
+# pylint: disable=C0302
```

Comment:
Better to use names instead of these codes

Reply:
ok